pod/perldelta.pod

```diff
@@ -69,6 +69,31 @@ They were deprecated in Perl v5.38.0 and scheduled for removal in Perl
 
 These features have now been entirely removed.
 
+If you code currently uses C<given>/C<when> or smart match this can be
+replaced with an C<if>/C<else> chain, or there is a variety of
```

Comment:
I think `is` is technically right here, but `are a variety` sounds better to me

Reply:
I originally used "are", but I've dodged the problem now.